libqtile/backend/x11/window.py

```diff
@@ -1694,7 +1694,7 @@ def __init__(self, window, qtile):
         self._wm_class: list[str] | None = None
         self.update_wm_class()
         self.update_name()
-        self.set_group()
+        self.togroup()
```

Comment:
I appreciate you're doing this for `Systray` but this will impact every window created this way. Every window would end up ignoring requests from the server to be placed on a different group or, when it's a transient, displaying with its parent.